R/StanModule.R

```diff
@@ -144,11 +144,13 @@ setMethod(
         stan_fragments <- lapply(
             names(stan_blocks),
             \(par) {
-                new_string <- c(slot(x, par), slot(y, par))
-                if (all(new_string == "")) {
-                    return("")
+                if (all(slot(y, par) == "")) {
```

Comment:
do you already have some unit tests for this specific thing? would it be worth to factor this into an explicit helper function outside of the `lapply()` to be able to test it better?